### PR TITLE
Updated to include local functions

### LINKED ISSUE #108: AV1720: Include local functions
### Existing rule:
**`Name methods using a verb or a verb-object pair (AV1720)`**
`Name methods using a verb like Show or a verb-object pair such as ShowDialog. A good name should give a hint on the what of a member, and if possible, the why.`
`Also, don't include And in the name of a method. It implies that the method is doing more than one thing, which violates the single responsibility principle explained in AV1115.`

### New language features to consider:
* local functions
I think this rule applies to local functions as well.

Proposal:
* Replace "methods" with "methods and/or local functions" (in title and text)
* Convert the text-reference to AV1115 into a hyperlink

### PR DESCRIPTION
**Pull request by bkoelman**
Fixes #108.

### CODE REVIEW

File: Src/Cheatsheet/Cheatsheet.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ NOTE: Requires Markdown Extra. See http://michelf.ca/projects/php-markdown/extra
 * Name types using nouns, noun phrases or adjective phrases (AV1708)
 * Don’t repeat the name of a class or enumeration in its members (AV1710)
 * Avoid short names or names that can be mistaken with other names (AV1712)
-* Name methods using verb-object pair (AV1720)
+* Name methods and local functions using verbs or verb-object pairs (AV1720)
```
Comment (dennisdoomen) [inline Src/Cheatsheet/Cheatsheet.md]: Hmm, when would a local function be named using just a `verb`?

File: Src/Guidelines/1700_NamingGuidelines.md [hunk 2]
```diff
@@ -94,10 +94,10 @@ Although technically correct, statements like the following can be confusing:
...
 
-### <a name="av1720"></a> Name methods using a verb or a verb-object pair  (AV1720) ![](images/2.png)
-Name methods using a verb like `Show` or a verb-object pair such as `ShowDialog`. A good name should give a hint on the *what* of a member, and if possible, the *why*.
+### <a name="av1720"></a> Name methods and local functions using verbs or verb-object pairs  (AV1720) ![](images/2.png)
+Name a method or local function using a verb like `Show` or a verb-object pair such as `ShowDialog`. A good name should give a hint on the *what* of a member, and if possible, the *why*.
 
-Also, don't include `And` in the name of a method. It implies that the method is doing more than one thing, which violates the single responsibility principle explained in AV1115.
+Also, don't include `And` in the name of a method or local function. That implies that it is doing more than one thing, which violates the single responsibility principle explained in AV1115.
```
Comment (dennisdoomen) [inline Src/Guidelines/1700_NamingGuidelines.md]: Also capitalize the SRP.

File: Src/Cheatsheet/Cheatsheet.md (hunk 1 above)
Comment (bkoelman) [reply Src/Cheatsheet/Cheatsheet.md]: I consider LFs similar to methods in that they perform an operation. Examples would then be: Initialize/InitializeFields, Prepare/PrepareDocument, Calculate/CalculateSalesRates, Reassign/ReassignWork etc.

File: Src/Guidelines/1700_NamingGuidelines.md (hunk 2 above)
Comment (bkoelman) [reply Src/Guidelines/1700_NamingGuidelines.md]: Sure, will update.

**Pull request merged**